samza-core/src/main/java/org/apache/samza/runtime/ApplicationRunnerUtil.java

```diff
@@ -40,6 +41,7 @@ public class ApplicationRunnerUtil {
    */
   public static ApplicationRunner invoke(Config originalConfig, ApplicationRunnerOperation op) {
     Config config = ConfigUtil.rewriteConfig(originalConfig);
+    ZkSystemEnvironmentSetter.setZkEnvironment(config);
```

Comment:
Can we please log all the system properties used by the application runner during the startup phase, since System properties is used to  set the ZK properties. It will be helpful for debugging purposes.

Reply:
will update